### DISCUSSION TITLE
Are beam weapons targeting worse in 0.10.0?

### DISCUSSION THREAD

**Discussion by IsocratesGrey** [endless-sky/endless-sky#8509] [General]
I started a new playthrough for 0.10.0 and I'm finding that beam weapons are targeting worse than they were before, I'm finding that they are trying to lead moving ships and are leading them too much so that they are completely missing many fighters. I didn't want to put it in as an issue because it may be intended. Is anyone else noticing the same?

**Comment by bene-dictator** [comment]
I had a similar problem, and apparently it's to do with projectile weapons. The game tries to let you hit both, but it isn't able to, and has the beam overshoot. Something like that.

**Comment by IsocratesGrey** [comment]
I have been unable to replicate the problem. I tried a number of ways so it must have been something very specific to what was happening. Alternatively, my bug finding instance is a direct build of 0.10.0 and that playthrough with the issue was on my steam game.
